docs/guides/react.md

```diff
@@ -0,0 +1,61 @@
+# video.js and ReactJS integration
+
+Here's a basic ReactJS player implementation.
+
+It just instantiate the video.js player on `componentDidMount` and destroy it on `componentWillUnmount`.
+
+```jsx
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import videojs from 'video.js'
+
+// this loads video.js CSS using webpack loaders
+require('!style-loader!css-loader!video.js/dist/video-js.css')
```

Comment:
I think we may want to exclude this line and the comment above as it is more webpack specific